administrator/components/com_associations/views/association/tmpl/edit.php

```diff
@@ -51,7 +52,20 @@
 	Joomla.loadingLayer('load');
 
 	// Attach behaviour to toggle button.
-	$('body').on('click', '#toogle-left-panel', function() {
+	$('body').on('click', '#toogle-left-panel', function()
+	{
+		referenceHide = $(this).attr('data-hide-reference');
+		referenceShow = $(this).attr('data-show-reference');
```

Comment:
vanilla js...
referenceHide = this.getAttibute('data-hide-reference');
eferenceShow = this.getAttibute('data-show-reference');